src/pykeen/sampling/negative_sampler.py

```diff
@@ -21,7 +21,9 @@ class NegativeSampler(ABC):
     """A negative sampler."""
 
     #: The default strategy for optimizing the negative sampler's hyper-parameters
-    hpo_default: ClassVar[Mapping[str, Mapping[str, Any]]]
+    hpo_default: ClassVar[Mapping[str, Mapping[str, Any]]] = dict(
+        num_negs_per_pos=dict(type=int, low=1, high=100, q=10),
```

Comment:
This leads to warnings since `low % q !=  0`. A simple fix would be to set `q=1`. This makes the search space larger (but the `q=1` seems to be quite arbitrary anyway 🤷 ).

Reply:
we have a log scale thing right? I think `1, 10, 100` is a good search space

Reply:
From my point of view, a log-scale, but integer search space would be appropriate. I am not sure whether we need to ensure that this number is divisible by `q>1` though.

Reply:
I'm going to address this in a separate PR #446 before finishing this one